polaris-react/src/components/DataTable/DataTable.tsx

```diff
@@ -425,46 +434,62 @@ class DataTableInner extends PureComponent<CombinedProps, DataTableState> {
     };
   };
 
-  private stickyHeaderScrolling = (event: Event) => {
+  private focusListener = (event: Event) => {
```

Comment:
Unecessary indirection? WDYT about calling `stickyHeaderFocusStateChangeOnScroll` directly?

Reply:
True, I might be early-optimizing